packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -884,4 +969,124 @@ test.describe('PieTextarea - Component tests', () => {
             });
         });
     });
+
+    test.describe('Attributes', () => {
+        test.describe('aria-describedby', () => {
+            test.describe('when `assistiveText` is NOT defined', () => {
+                test('should not render the attribute', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieTextarea, {
+                        props: {} as TextareaProps,
+                    });
+
+                    // Act
+                    const textarea = component.locator('textarea');
+
+                    const componentAttribute = await textarea.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBeNull();
+                });
+            });
+
+            test.describe('when `assistiveText` is defined', () => {
+                test('should render the attribute correctly with the correct value', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieTextarea, {
+                        props: {
+                            assistiveText: 'Some useful message',
+                        } as TextareaProps,
+                    });
+
+                    // Act
+                    const textarea = component.locator('textarea');
+
+                    const componentAttribute = await textarea.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBe('assistive-text');
+                });
+            });
+        });
+
+        test.describe('aria-invalid', () => {
+            test.describe('when the component status is set to `error`', () => {
+                test('should render the `aria-invalid` attribute', async ({ mount }) => {
```

Comment:
Isn't this attribute is rendered regardless, with a value of "true" or "false"?

Reply:
This test can probably just be updated to check for the "true" value, because the test below checks the "false" cases